sdk/core/Azure.Core/samples/ProtocolMethods.md

```diff
@@ -92,65 +92,55 @@ Protocol methods need a JSON object of the shape required by the schema of the s
 See the specific service documentation for details, but as an example:
 
 ```csharp
-	var data = new {
-                name = "Buddy",
-                id = 2,
-                color = "Brown"
-        };
-        /*
-        {
-            "name": "Buddy",
-            "id": 2,
-            "color": "Brown"
-        }
-        */
-        client.SetDog(RequestContent.Create(data));
+var data = new {
+    name = "Buddy",
+    id = 2,
+    color = "Brown"
+};
+/*
+{
+    "name": "Buddy",
+    "id": 2,
+    "color": "Brown"
+}
+*/
+client.SetDog(RequestContent.Create(data));
 ```
 
 ## 3. Handle the Response
 
-Protocol methods all return a `Response` object that contains information returned from the service request. 
+Protocol methods all return a `Response` object that contains information returned from the service request.
 
 The most important field on Response contains the REST content returned from the service:
 
 ```csharp
-        Response response = client.GetDog(RequestContent.Create(new {
-                name = "Buddy"
-        }));
-        var content = response.Content;
+Response response = client.GetDog(RequestContent.Create(new {
```

Comment:
I assume we are using an anonymous type and relying on the default STJ serializer to serialize to JSON? Can we explain that because I don't think it is obvious from the usage alone.

Reply:
explanation added